src/Raven.Studio/typescript/viewmodels/database/query/spatialQueryMap.ts

```diff
@@ -0,0 +1,104 @@
+import viewModelBase = require("viewmodels/viewModelBase");
+import spatialMapLayerModel = require("models/database/query/spatialMapLayerModel");
+import document = require("models/database/documents/document");
+import documentMetadata = require("models/database/documents/documentMetadata");
+import {Layer, MarkerClusterGroup} from "leaflet";
+
+class spatialQueryMap extends viewModelBase {
+    
+    geoData = ko.observableArray<spatialMapLayerModel>([]);
+
+    constructor(geoData: Array<spatialMapLayerModel>) {
+        super();
+        this.geoData(geoData);
+    }
+
+    compositionComplete() {
+        super.compositionComplete();
+        this.createMap();
+    }
+    
+    createMap() {
+
+        ////////////////////////////
```

Comment:
please remove those comment banners - either use single line comments:
```
// Define base tile layers
```

or create separate methods and call them one by one.

Reply:
let's keep coding style consistent.

Reply:
done